modules/apps/frontend-css/frontend-css-variables-web/src/test/resources/com/liferay/frontend/css/variables/web/internal/theme/liferay-look-and-feel.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!DOCTYPE look-and-feel PUBLIC "-//Liferay//DTD Look and Feel 7.0.0//EN" "http://www.liferay.com/dtd/liferay-look-and-feel_7_0_0.dtd">
+
+<look-and-feel>
```

Comment:
Does this file need to be so "complete"?

Since it's for a mock... could we just leave here the least amount of necessary information? I got pretty scared for a while thinking we had to add a whole theme or something for this...

Also, it'd help to make this explicit if we did something like `theme id="testTheme" name="Test Theme"`.

![just-sayin](https://user-images.githubusercontent.com/905006/85576788-fe5aaa80-b638-11ea-89c6-37281fc3b123.gif)

Reply:
OK. Sounds good. :+1: